src/ray/protobuf/experimental/autoscaler.proto

```diff
@@ -190,6 +190,24 @@ message PendingInstanceRequest {
   string ray_node_type_name = 2;
   // The number of such instance requested to be started.
   int32 count = 3;
+  // The unix timestamp when the request is made.
+  int64 request_ts = 4;
+}
+
+message FailedInstanceRequest {
+  // The instance type name for the cloud, e.g. t1.micro
+  string instance_type_name = 1;
+  // The node type name, e.g. ray-head-node, matching `available_node_types`
+  // in the autoscaler config. See `ray/autoscaler/ray-schema.json`
+  string ray_node_type_name = 2;
+  // The number of such instance requested to be started.
+  int32 count = 3;
+  // Descriptive reasons for the failure.
+  string reason = 4;
```

Comment:
fyi, for reason, the msg can be long in size.

Reply:
This is fine?

Reply:
yup, just fyi